releases/tp1_release.md

```diff
@@ -0,0 +1,74 @@
+# Release Syndesis TP1
+
+This is the release plan for TP1, which consist of multiple distinct repos. This plan changes when going to a Monorepo, as many of the projects below then share the same version number (and releasy cycle)
+
+## Tagging and Building
+
+### External Images
+
+* https://github.com/syndesisio/pemtokeystore
```

Comment:
Not need for TP2 though

Reply:
left it here for historical reason, removed from TP2 document.